### DISCUSSION TITLE
Custom icons for actions dropdown

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#301] [Feature requests]
Currently, the actions dropdown features a play icon for each action. We might be able to give the users the ability to configure them per action.
![image](https://user-images.githubusercontent.com/1334409/113413355-33174180-93c3-11eb-8184-8a2e88b5ecc9.png)

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/actions/registration.html#icon
